v6README.md

```diff
@@ -47,6 +49,22 @@ yarn add @storybook/react-native@next \
             @storybook/addon-controls
 ```
 
+**Expo**
+
+For Expo, we need to separately install the react native packages so Expo can maintain the compatibility for us.
+
+```shell
+yarn add @storybook/react-native@next \
```

Comment:
I think you could list all the dependencies with expo install, I believe it will just do a regular yarn install for packages without native dependencies. Something like this:

```
expo install @storybook/react-native@next \
            @storybook/addon-ondevice-actions@next \
            @storybook/addon-ondevice-controls@next \
            @storybook/addon-ondevice-backgrounds@next \
            @storybook/addon-ondevice-notes@next \
            @storybook/addon-actions \
            @storybook/addon-controls \
            @react-native-async-storage/async-storage \
            @react-native-community/datetimepicker \
            @react-native-community/slider
```